### PR TITLE
[ios] Remove traffic button from CarPlay 

### LINKED ISSUE #868: Remove not working traffic button artifact from the CarPlay

### PR DESCRIPTION
**Pull request by fabwu**
The traffic button is not working in CarPlay, so this commit removes it to not confuse users.

Close #868

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -22527,7 +22527,7 @@
     zh-Hant = 使用條款
 
   [button_layer_traffic]
-    tags = android,ios
+    tags = android
```
Comment (biodranik) [inline data/strings/strings.txt:22530]: Does it mean that Android still has a traffic button somewhere? Can/should it also be removed?
CC @Jean-BaptisteC

**Review by biodranik — APPROVED** [review]
Thanks!

File: data/strings/strings.txt (hunk 1 above)
Comment (Jean-BaptisteC) [reply data/strings/strings.txt:22530]: Button is not present on Android but code base was keep

Comment (fabwu) [reply data/strings/strings.txt:22530]: Ok I thought I remove the code. It can always be recovered from git. But you guys decide how you want it.

Comment (fabwu) [reply data/strings/strings.txt:22530]: @biodranik I think this is ready to merge

**Comment by kirylkaveryn** [comment]
@fabwu can you please add the before/after screenshots if possible?

**Comment by fabwu** [comment]
> @fabwu thanks a lot! Can you please add the before/after screenshots if possible?

Before:

Screenshot 2024-06-22 at 14 44 39

After:

Screenshot 2024-06-22 at 14 46 04

**Review by biodranik — APPROVED** [review]
Thank you, Fabian!

**Pull request merged**